.github/workflows/sonarcloud.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   build:
+    if: github.repository_owner == 'damianszczepanik'
```

Comment:
I like this one, can you move it to separate branch and merge with comment https://github.com/damianszczepanik/cucumber-reporting/commit/d6281b44a197d7f2653ebe33cfe66e9e7130c31a so we can test it?